### PR TITLE
feat(server): distinguash workspace quota calc

### PR DESCRIPTION
**Pull request by darkskygit**
fix AF-2021

### CODE REVIEW

**Comment by darkskygit** [comment]
* **#9338** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9338?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9338?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `canary`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/toeverything?ref=merge-queue-instructions-in-stack-comment&prId=9338).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/backend/server/src/core/quota/storage.ts [hunk 1]
```diff
@@ -98,6 +98,10 @@ export class QuotaManagementService {
     }, 0);
   }
 
+  async getWorkspaceStorageUsage(workspaceId: string) {
```
Comment (Brooooooklyn) [inline packages/backend/server/src/core/quota/storage.ts:106]: why this wrapper is needed

**Review by Brooooooklyn — COMMENTED** [review]
Need tests

File: packages/backend/server/src/core/quota/storage.ts (hunk 1 above)
Comment (darkskygit) [reply packages/backend/server/src/core/quota/storage.ts:106]: code updated, ensure that size is within the safe range of gql

File: packages/backend/server/tests/quota.spec.ts [hunk 2]
```diff
@@ -165,3 +162,33 @@ test('should be able to override quota', async t => {
   t.is(wq3.storageQuota, 140 * OneGB, 'should be override to 120GB');
   t.is(wq3.memberLimit, 2, 'should be override to 1');
 });
+
+test.only('should be able to check with workspace quota', async t => {
```
Comment (Brooooooklyn) [inline packages/backend/server/tests/quota.spec.ts]: remove `.only`

**Review by forehalo — APPROVED** [review]

**Pull request merged**